app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PetApplicationsController < ApplicationController
+  def new
+    @favorite_pets = favorite.pets
+  end
+
+  def create
+    application = PetApplication.new(
+      name: params[:name],
+      address: params[:address],
+      city: params[:city],
+      state: params[:state],
+      zip: params[:zip],
+      phone_number: params[:phone_number],
+      description: params[:description],
+      pet_ids: params[:pet_ids]
+    )
```

Comment:
This controller action is pretty long. We could use a private method to split this up.